instructure/canvas-ios
# Module list item download

refs: MBL-17137
affects: Student
release note: none
test plan: 
1. Login as tamaskozmer 
2. Sync course "course with no tabs" 
3. Check if module subitems are downloaded 
4. Check an other account/regular course if everything still works. 

## Checklist
